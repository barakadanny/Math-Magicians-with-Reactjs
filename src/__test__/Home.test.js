import React from 'react';
import renderer from 'react-test-renderer';
import Home from '../components/Home';
import '@testing-library/jest-dom';

describe('Test Home page', () => {
  it('testing Home page UI rendering', () => {
    const tree = renderer.create(<Home />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
