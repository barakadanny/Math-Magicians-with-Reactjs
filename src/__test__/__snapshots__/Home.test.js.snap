// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Home page testing Home page UI rendering 1`] = `
<div
  className="container home"
>
  <h1
    className="title"
  >
    Welcome to Math Magician
  </h1>
  <p>
    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
  </p>
</div>
`;
